charts/cronjob/templates/cronjob.yaml

```diff
@@ -20,6 +20,10 @@ spec:
           containers:
             - name: "{{ .Values.cronJobName }}"
               image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              {{- if .Values.args }}
+              args:
+              - {{ .Values.args }}
```

Comment:
Sollte in der `values.yaml` dokumentiert sein

Reply:
stimmt!